lib/renderer/api/desktop-capturer.ts

```diff
@@ -20,20 +14,5 @@ function getCurrentStack () {
 }
 
 export async function getSources (options: Electron.SourcesOptions) {
-  if (!isValid(options)) throw new Error('Invalid options');
-
-  const captureWindow = options.types.includes('window');
-  const captureScreen = options.types.includes('screen');
-
-  const { thumbnailSize = { width: 150, height: 150 } } = options;
-  const { fetchWindowIcons = false } = options;
-
-  const sources = await ipcRendererInternal.invoke<ElectronInternal.GetSourcesResult[]>('ELECTRON_BROWSER_DESKTOP_CAPTURER_GET_SOURCES', {
-    captureWindow,
-    captureScreen,
-    thumbnailSize,
-    fetchWindowIcons
-  } as ElectronInternal.GetSourcesOptions, getCurrentStack());
-
-  return deserialize(sources);
+  return deserialize(await ipcRendererInternal.invoke('ELECTRON_BROWSER_DESKTOP_CAPTURER_GET_SOURCES', options, getCurrentStack()));
```

Comment:
Not sure if this would look better in the release build, but I wish we didn't expose the internal IPC name so prominently in the error. Probably not critical for this PR, but maybe something we could improve broadly in the future.

```
Uncaught (in promise) Error: Error invoking remote method 'ELECTRON_BROWSER_DESKTOP_CAPTURER_GET_SOURCES': Error: Invalid options
    at EventEmitter../lib/renderer/ipc-renderer-internal.ts.exports.ipcRendererInternal.invoke (ipc-renderer-internal.ts:29)
    at async Object.getSources (desktop-capturer.ts:17)
```